app/components/roadmap-page/-utils/code-splitting.yaml

```diff
@@ -57,6 +57,9 @@
     - type: rfc
       name: Packaging Strategies RFC
       url: https://github.com/ember-cli/rfcs/pull/110
+      #- type: quest
+      #- name: Quest Issue
+      #- url: https://github.com/ember-cli/ember-cli/issues/4211
```

Comment:
We decided to comment these out for now as they aren't current but are links we intend to refresh as new quest issues come in.